js/components/controlBar.js

```diff
@@ -727,14 +701,37 @@ var ControlBar = createReactClass({
               null
           }
           height={this.props.skinConfig.controlBar.logo.height} />
+      ),
+
+      skipControls: (
+        <SkipControls
+          key={CONSTANTS.CONTROL_BAR_KEYS.SKIP_CONTROLS}
+          audioOnly={this.props.audioOnly}
```

Comment:
I haven't seen this part of the code yet, but would it make more sense to make the component configurable as to meet audio only requirements instead of making it aware of audio only logic?

Reply:
Ah, yep, there are a few things that this could be applied to. I missed this when cleaning up the code. I'll add these new configs in. However, we may still need the audioOnly prop to retrieve the audio only button config. Do you know of a way to do this otherwise?

Reply:
Yes so definitely at some point we'll need the audioOnly check, I just think it might be better at a higher level. The button config could be passed to the component as a prop (instead of it getting it directly from the skin config) that way the check happens earlier and the SkipControls don't need to know about audio only

Reply:
Agreed, will make these changes